spiffworkflow-backend/src/spiffworkflow_backend/routes/openid_blueprint/openid_blueprint.py

```diff
@@ -114,13 +114,12 @@ def token() -> Response | dict:
     authorization = base64.b64decode(authorization).decode("utf-8")
     client_id = authorization.split(":")
 
-    host_url = current_app.config.get("SPIFFWORKFLOW_BACKEND_URL", request.host_url.strip("/"))
-    base_url = f"{host_url}/openid"
```

Comment:
I removed the intermediate `base_url` here because I wanted the token's issuer value to exactly match the issuer specified in line 47 even under casual observation.